lang/cs/Org.Apache.REEF.Client/API/JobRequestBuilderFactory.cs

```diff
@@ -27,13 +27,12 @@ namespace Org.Apache.REEF.Client.API
     /// </summary>
     internal sealed class JobRequestBuilderFactory
     {
-        private readonly ISet<IConfigurationProvider> _configurationProviders;
+        private readonly IInjector _injector;
 
         [Inject]
-        private JobRequestBuilderFactory(
-            [Parameter(typeof(DriverConfigurationProviders))] ISet<IConfigurationProvider> configurationProviders)
+        private JobRequestBuilderFactory(IInjector injector)
```

Comment:
Because as written we would need to inject everything into `JobRequestBuilderFactory` that we want to inject into `JobRequestBuilder`.

I added a new parameter to be injected in `JobRequestBuilder`, and rather than propagate that request to the `Factory`, I simply have the `Factory` ask for a handle on the Injector.